picard/ui/util.py

```diff
@@ -91,3 +92,82 @@ def __init__(self, *args):
         for view in self.findChildren((QtWidgets.QListView, QtWidgets.QTreeView)):
             if isinstance(view.model(), QtWidgets.QFileSystemModel):
                 view.setSelectionMode(QtWidgets.QAbstractItemView.ExtendedSelection)
+
+
+# Theme
+
+
+class DefaultThemeColor:
```

Comment:
I would move the theme / color related classes to a separate file, e.g. `ui/colors.py`. It's a bit more then what we usually have in utils files.

Reply:
Yep, agreed.